state-chain/pallets/cf-cfe-interface/src/lib.rs

```diff
@@ -37,9 +36,29 @@ pub type TxBroadcastRequest<T, C> =
 pub mod pallet {
 
 	use cf_traits::Chainflip;
+	use frame_support::pallet_prelude::*;
 
 	use super::*;
 
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config> {
+		pub cfe_events_version: u32,
```

Comment:
I don't think this needs to be a parameter, we can just set it to CFE_EVENTS_VERSION